src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_header">
+      <div class="moyo_header_logo">
+        <a
+          href="#"
+          class="menu_link"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+      <nav>
+        <ul class="moyo_menu">
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Video
+            </a>
+          </li>
```

Comment:
There should be an empty line between sibling elements that are blocks. Add an empty line between the `li` elements.